tests/Unit/Validator/Constraints/RouteDefaultsValidatorTest.php

```diff
@@ -12,60 +12,122 @@
 namespace Symfony\Cmf\Bundle\RoutingBundle\Tests\Unit\Validator\Constraints;
 
 use Symfony\Cmf\Bundle\RoutingBundle\Validator\Constraints\RouteDefaults;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
 use Symfony\Component\Validator\Tests\Constraints\AbstractConstraintValidatorTest;
 
-abstract class RouteDefaultsValidatorTest extends AbstractConstraintValidatorTest
```

Comment:
are the two doing the same? then we could instead do a block above this that does

```
if (!class_exists(ContraintValidatorTestCase::class)) {
    abstract class ContraintValidatorTestCase extends AbstractConstraintValidatorTest {}
}
```

then the code below could stay untouched. if our code is is different between the two cases, this won't work however.

Reply:
Ah this would work? Tried to create conditional use statements with an equal alias, this did not work.

Reply:
@dbu can you push the fix please? I won't be on my PC before tonight.